documentation/bibliography.bib

```diff
@@ -956,3 +956,19 @@ @unpublished{Celledoni2007
   title  = {The exact computation of the free rigid body motion and its use in splitting methods},
 }
 
+@book{Brent1973,
+  author    = {Brent, Richard P.},
+  publisher = {Prentice-Hall},
+  date      = {1973},
+  title     = {Algorithms for minimization without derivatives},
```

Comment:
A book should have an ISBN, unless it was written by a monk in an abbey in the 13th century.